src/sentry/receivers/onboarding.py

```diff
@@ -20,6 +24,23 @@
 from sentry.utils.javascript import has_sourcemap
 
 
+def check_for_onboarding_complete(organization):
```

Comment:
It's coming from `event_processed` signal, so actually, it's possible that this is called a few times in a single event, since there are multiple receivers for this signal.